django/api/services/spreadsheet_uploader.py

```diff
@@ -185,17 +195,20 @@ def import_from_xls(
         if check_for_warnings:
             ## do the error checking
             typo_warnings = typo_checker(df, df['applicant_name'].dropna(), .8)
-            if typo_warnings:
+
+            validation_errors['typo_warnings'] = typo_warnings
```

Comment:
we wouldn't need this line and we also wouldn't need line 197 if we use the `validation_functions` structure in the comment above.